js/views/playingScreen.js

```diff
@@ -115,16 +121,94 @@ var PlayingScreen = React.createClass({
       this.showControlBar(event);
       this.props.controller.startHideControlBarTimer();
     }
-    else {
-      this.props.controller.togglePlayPause();
+    else if (!this.isVideo360) {
+      this.props.controller.togglePlayPause(event);
+    }
```

Comment:
Since we have a new flow now for detecting a player click to trigger togglePlayPause, can this flow be used in the non-vr 360 path? If so, we can remove this else if statement and simplify the code.